docs/integrations/common-workflow-steps/firebase-upload-dsym.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Firebase Upload dSYM 
+metaTitle: Firebase Upload dSYM
+metaDescription: Firebase Upload dSYM
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Firebase Upload dSYM
+
+This step allows to to upload your debug symbols to Firebase Crashlytics. 
+
+### Prerequisites
+
+This step depends on the **Xcodebuild for Devices** step to upload the .dSYM file.
```

Comment:
This is not an error but if you want to avoid repetition, you may not write here what you wrote in the table. You can refer to the table for information as below.
```
The workflow steps that need to be executed before running the `Firebase Upload dSYM` workflow step, along with their respective reasons, are listed in the table below.
```